package.json

```diff
@@ -79,10 +79,10 @@
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "7.1.1",
     "@schematics/angular": "^12.0.0",
-    "@storybook/addon-knobs": "^6.2.7",
-    "@storybook/angular": "^6.2.7",
+    "@storybook/addon-essentials": "^6.2.9",
```

Comment:
I think it's good to add the `addon-essentials` as I've seen that being added in the default storybook setup as well. Do we know what is all part of it?

Reply:
Also, we should not use `^` in the versions for Storybook, but rather `~` since we don't want to upgrade feature version numbers as they might be breaking (as we've seen).

I'm addressing the main storybook version changes here as well: https://github.com/nrwl/nx/pull/6104

Reply:
The storybook ppl suggested we could use the `essentials` addon, so I thought I just might! hehe, there's a bunch of things and users can disable things they don't want in `main.js` I think

Reply:
eg https://storybook.js.org/docs/riot/essentials/introduction#disabling-addons